addition.js

```diff
@@ -0,0 +1,12 @@
+function s(a,b){
```

Comment:
Uvek stavi neka smislena imana za funkcije, pogledaj 
https://github.com/ryanmcdermott/clean-code-javascript#function-names-should-say-what-they-do